docs/setup/new_cluster.md

```diff
@@ -0,0 +1,695 @@
+      /opt/shared_cluster. And the directory name is based on "shared_" +
+      the name of the CommCare HQ environment. CommCare HQ environments
+      are usually named after their organization or project. For this
+      example we will name it "cluster".
+
+   2. Attach the volumes to the VMs:
+
+            $ virsh attach-disk --domain db1 \
+                --source /srv/virt/cchq-cluster/images/db1_data.qcow2 \
+                --target vdb \
+                --config --live
+            $ virsh attach-disk --domain db2 \
+                --source /srv/virt/cchq-cluster/images/db2_data.qcow2 \
+                --target vdb \
+                --config --live
+            $ virsh attach-disk --domain db2 \
+                --source /srv/virt/cchq-cluster/images/db2_shared_cluster.qcow2 \
+                --target vdc \
+                --config --live
+
+   3. Ensure the VMs can use their full size:
+
+            $ virsh blockresize --domain db1 \
+                --path /srv/virt/cchq-cluster/images/db1_data.qcow2 \
+                --size 60G
+            $ virsh blockresize --domain db2 \
+                --path /srv/virt/cchq-cluster/images/db2_data.qcow2 \
+                --size 60G
+            $ virsh blockresize --domain db1 \
+                --path /srv/virt/cchq-cluster/images/db2_shared_cluster.qcow2 \
+                --size 20G
+
+   4. Prepare the disks on the VMs.
+
+      Repeat the following steps for /dev/vdb (/opt/data) on db1, and
+      /dev/vdb (/opt/data) and /dev/vdc (/opt/shared_cluster) on db2:
+
+      1. Log into the VM as "ansible". e.g.:
+
+                $ ssh ansible@db1
+
+       2. Partition and format the disk. e.g.:
+
+                $ sudo fdisk /dev/vdb
+                ...
+                $ sudo mkfs.ext4 /dev/vdb1
+
+        3. Add the disk to /etc/fstab. Find its UUID using `blkid`, and
+           then append an entry for its mount point to /etc/fstab. e.g.:
+
+                $ sudo blkid
+                ...
+                /dev/vdb1: UUID="11111111-2222-3333-4444-555555555555" ...
+
+                $ echo 'UUID=11111111-2222-3333-4444-555555555555  /opt/data  ext4  defaults  0 0' | sudo tee -a /etc/fstab
+
+                $ sudo mkdir /opt/data
+
+                $ sudo mount -a /opt/data
+
+           Of course, for /dev/vdc on db2, the mount point will be
+           /opt/shared_cluster instead of /opt/data.
+
+6. Increase the memory for webworker1, webworker2, db1, and db2. Do the
+   following for each VM:
+
+   1. Shut the VM down. e.g. For webworker1:
+
+            $ virsh shutdown webworker1
+
+   2. Increase the memory.
+
+            $ virsh setmaxmem webworker1 8G --config
+            $ virsh setmem webworker1 8G --config
+
+   3. Restart the VM.
+
+            $ virsh start webworker1
+
+Configuration of your virtual machines is complete. Adapt the
+instructions above for your requirements and your virtualization stack.
+
+
+Prepare all VMs for automated deploy
```

Comment:
From here downwards seems very similar to the monolith installation docs. Maybe we can reference that, and just point out what is different for a cluster?

Reply:
Referencing the monolith instructions could be informative. But I'd like to push back against "just point out what is different". I don't think that is a more reader-friendly approach.

In fact, I would offer our HQ [DEV_SETUP](https://github.com/dimagi/commcare-hq/blob/master/DEV_SETUP.md) documentation as a good example of what _not_ to do.

Here is how Slack does it: First you [choose your OS](https://slack.com/intl/en-gb/help/categories/360000049043-Getting-started#download-slack). Then, if you choose Linux (for example) there are similar, but separate, [instructions for each distro](https://slack.com/intl/en-gb/help/articles/212924728-Download-Slack-for-Linux--beta-#ubuntu-1). You can imagine how unreadable those instructions would be if Slack had taken our approach:

> ...
> Step 4:
> 1. (Ubuntu) Double-click the file to view Slack in Ubuntu Software Centre.
> 2. (Fedora) Open the file in your package manager.
> 3. (RHEL) Open Terminal and run the following commands ...
>
> ...

DEV_SETUP should start off with by linking to the right section for your operating system, and then present only the instructions applicable to you. Sure, there will be a lot of duplication. But it would be a whole lot more readable.

Readability is what I hoped to achieve with the structure I chose here.

Reply:
OK, but I think the only difference here is how you set up the inventory file, right? This seems like a lot of overhead to manage two of the same document when I think this could just be a subsection of "setting up a new installation". It would then be like the tabs in slack.

Reply:
Hmmm. That sounds fair.

I'll merge them into a "setting up a new installation" doc.

Reply:
@proteusvacuum [a20e336](https://github.com/dimagi/commcare-cloud/pull/5015/commits/a20e336e4354bc4e8ea64853c569c5b23d3e2769)

Probably easier to review the [merged doc as rendered](https://github.com/dimagi/commcare-cloud/blob/a20e336e4354bc4e8ea64853c569c5b23d3e2769/docs/setup/new_environment.md).